### DISCUSSION TITLE
Choosing between audio input and output

### DISCUSSION THREAD

**Discussion by GuckicheLorraine** [TheWisker/Cavasik#8] [Q&A]
How do you choose between audio input and output for cavasik ?
Last night it was stuck on my output to have the visualizers and today it is stuck on my input.

**Comment by TheWisker** [comment]
it's supposed to be automatic, it selects from pulseaudio.

**Comment by TheWisker** [comment]
Anyway, as far as I know you can select which input Cavasik receives from PulseAudio with programs like pavucontrol
